packages/client/hmi-client/src/workflow/ops/simulate-ensemble-ciemss/tera-simulate-ensemble-ciemss.vue

```diff
@@ -43,7 +43,7 @@
 									<th>Weight</th>
 								</thead>
 								<tbody class="p-datatable-tbody">
-									<tr v-for="(id, i) in ensembleConfigs.map((ele) => ele.id)" :key="i">
+									<tr v-for="(id, i) in listModelLabels" :key="i">
```

Comment:
I am using index matching here. Im not sure if i should
add a frontend version of EnsembleModelConfigs that includes the name to avoid index matching

Im not a huge fan of index matching but also having a frontend extension of an existing object seems bleh too. thoughts?

Reply:
Maybe a computed variable could act as the frontend extension...not sure how this all works though. If it works for now maybe just save this for later